src/program.js

```diff
@@ -201,10 +205,18 @@ export class Program {
         });
       });
 
+      if (adjustedArgv.verbose) {
+        logStream.makeVerbose();
+
+        if (!versionLogged) {
+          log.info('Version:', getVersion(absolutePackageDir));
```

Comment:
Oof. This duplication is unfortunate. I can't think of another way to do it though. I suppose you could pull it into a shared function just for a bit more maintainability.

Reply:
yeah, I agree, I've refactored the part that enable the verbose mode and print the version into a class method and reused it into the two places where we have to enable the verbose mode (when it is specified on the command line and when it is only specified in a config file).

Let me know how it looks to you.